types/index.d.ts

```diff
@@ -0,0 +1,50 @@
+import type { FilePond as FilePondInstance, FilePondOptions } from "filepond";
+import type { SvelteComponentTyped } from "svelte";
+
+export * from "filepond";
+
+type ComponentProps = FilePondOptions & {
+  class?: string;
+  id?: string;
+  /** A string specifying the name of the input control. */
+  name: string;
+  /**
+   * The Boolean multiple attribute, if set, means the form control accepts one or more values.
+   * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/multiple
+   */
+  allowMultiple?: boolean;
+  /** 
+   * The Boolean required attribute, if present, indicates that the user must specify a value for the input before the owning form can be submitted. 
+   **/
+  required?: boolean;
+  /**
+   * The capture attribute specifies that, optionally, a new file should be captured, and which device should be used to capture that new media of a type defined by the accept attribute.
+   * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture
+   */
+  captureMethod?: 'user' | 'environment' | boolean;
+  /** 
+   * Comma-separated list of accepted file types
+   * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#attr-accept
+   **/
+  acceptedFileTypes?: string;
```

Comment:
This conflicts with the filepond-plugin-file-validate-type plugin, it says it should accept a `string[]` but this says it should be a `string`. The plugin already has TypeScript support so this line should be removed.